.ci/perf_tests/k6scripts/SyncHelloWorld.js

```diff
@@ -35,7 +35,7 @@ export let options = {
 
 // Main function
 export default function () {
-    let response = http.get(`${PROTOCOL}://${HOSTNAME}:${PORT}/api/SyncHttpTriggerHelloWorld`);
+    let response = http.get(`${PROTOCOL}://${HOSTNAME}:${PORT}/api/SyncHelloWorld`);
```

Comment:
Make the scenario as a configuration variable Such that other scenarios can work.

Reply:
this client file is only for calling SyncHelloWorld endpoint, not sure why we want to extend this to other cases? we have one client file per endpoint 
<img width="240" alt="image" src="https://github.com/Azure/azure-functions-python-worker/assets/12449837/924d98e1-413d-4c44-af70-94a1ec17ce94">

Reply:
Makes sense. Thanks for clarifying